docker/gaffer-federated-store/.env

```diff
@@ -0,0 +1,2 @@
+GAFFER_VERSION=1.12.0
```

Comment:
Can you update Gaffer versions to the latest versions in line the rest of the repo please?

Reply:
Updated